src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form class="form-big" method="get" action="/">
+        <label for="bigSearch">
+          <input
+            class="keypress"
+            type="text"
+            name="search"
+            id="bigSearch"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="big"
+            data-qa="keypress"
```

Comment:
Use correct attributes. data-qa="big" is for the `form` tag, not for the `input`. Fix it, please 
<img width="987" alt="image" src="https://user-images.githubusercontent.com/99403198/234831891-17a171e7-0dec-42be-82ac-78e75b9eeb38.png">